eng/init-detect-binaries.proj

```diff
@@ -4,21 +4,33 @@
     <TargetFramework>$(NetCurrent)</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="$(ToolsDir)BinaryToolKit\BinaryToolKit.csproj" />
+  </ItemGroup>
+
+  <UsingTask TaskName="BinaryToolTask" AssemblyFile="$(BinaryToolKitAssembly)" TaskFactory="TaskHostFactory" />
+
+  <PropertyGroup>
+    <BinariesMode Condition="'$(BinariesMode)' == ''">Validate</BinariesMode>
+    <AllowedBinariesFile Condition="'$(AllowedBinariesFile)' == ''">$(RepoRoot)eng/allowed-vmr-binaries.txt</AllowedBinariesFile>
+  </PropertyGroup>
+
   <Target Name="DetectBinaries"
           BeforeTargets="Build"
```

Comment:
This makes it possible to just invoke this target locally by correctly defining a dependency on P2Ps getting built first.

```suggestion
          DependsOnTargets="ResolveProjectReferences"
          BeforeTargets="Build"
```